File: app/src/main/java/dag/iplogger/IpAddress.java
package dag.iplogger;

/**
 * Created by Dag on 15.12.2014.
 */
public class IpAddress {
    private String ipNo;
    private String dnsName;

    @Override
    public String toString() {
        return ipNo + "/" + dnsName;
    }

    public IpAddress(String ipNo, String dnsName) {
        this.ipNo = ipNo;
        this.dnsName = dnsName;
    }

    @Override
    public boolean equals(Object o) {
        return o != null && toString().equals(o.toString());
    }


}

File: app/src/main/java/dag/iplogger/IpAddress.kt
package dag.iplogger

/**
 * Created by Dag on 15.12.2014.
 */
data class IpAddress(private val ipNo: String, private val dnsName: String) {

    override fun toString(): String {
        return "$ipNo/$dnsName"
    }

    override fun equals(o: Any?): Boolean {
        return o != null && toString() == o.toString()
    }


}
